doc/user/content/sql/system-catalog.md

```diff
@@ -285,17 +286,33 @@ For per-worker frontier information, see
 
 Field       | Type       | Meaning
 ------------|------------|--------
-`global_id` | [`text`]   | The ID of the index that created the dataflow. Corresponds to [`mz_indexes.id`](#mz_indexes).
+`global_id` | [`text`]   | The ID of the index or materialized view that created the dataflow. Corresponds to [`mz_indexes.id`](#mz_indexes) or [`mz_materialized_views.id`](#mz_materialized_views).
 `time`      | [`bigint`] | The next timestamp at which the materialization may change.
 
+### `mz_materialization_source_frontiers`
+
+The `mz_materialization_source_frontiers` view describes the frontiers for every
+storage source used in a [dataflow] in the system across all workers. The frontier
+describes the earliest timestamp at which the output of the source instantiation
+at the dataflow layer may change; data prior to that timestamp is sealed.
+
+For per-worker frontier information, see
+[`mz_worker_materialization_source_frontiers`](#mz_worker_materialization_source_frontiers).
+
+Field       | Type       | Meaning
+------------|------------|--------
+`dataflow`  | [`text`]   | The ID of the index or materialized view that created the dataflow. Corresponds to [`mz_indexes.id`](#mz_indexes) or [`mz_materialized_views.id`](#mz_materialized_views).
+`source`    | [`text`]   | The ID of the input storage source for the dataflow. Corresponds to either [`mz_sources.id`](#mz_sources) or [`mz_tables.id`](#mz_tables) or [`mz_materialized_views.id`](#mz_materialized_views).
+`time`      | [`bigint`] | The next timestamp at which the source instantiation may change.
+
 ### `mz_materializations`
 
-The `mz_materializations` source describes the indexes in the system.
+The `mz_materializations` source describes the indexes and materialized views in the system.
 
 Field    | Type       | Meaning
 ---------|------------|--------
-`name`   | [`text`]   | The ID of the index. (`name` is a misnomer.)
-`worker` | [`bigint`] | The ID of the worker thread hosting the index.
+`name`   | [`text`]   | The ID of the index or materialized view. (`name` is a misnomer.)
+`worker` | [`bigint`] | The ID of the worker thread managing the corresponding [dataflow].
```

Comment:
I believe `hosting` is more specific, what do you think?

Reply:
I am fine with keeping hosting here; upon a second look, it aligns better with the rest of the documentation. I was a bit in doubt regarding the case of multiple workers being used for a given dataflow, but I suppose the worker referred here would be the one coordinating the process?